pikaur/main.py

```diff
@@ -132,12 +133,13 @@ def cli_info_packages(args):
 
 
 def cli_clean_packages_cache(args):
-    if os.path.exists(BUILD_CACHE):
-        print(f'\nBuild directory: {BUILD_CACHE}')
+    build_cache = os.path.join(CACHE_ROOT, BUILD_CACHE_DIR)
```

Comment:
missing `if isroot()` here

/var/cache/pikaur will be persistent with the current options in `isolate_root_cmd` func

Reply:
also i would like isroot to be is_root